notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -3087,6 +3087,48 @@ coping with the death of a parent [Kate grieved over the death of first her fian
 romantic love [John and Kate.]
 
 
+movie: Alien Hunter (2003)
+==========================
+
+:: Title
+Alien Hunter
+
+:: Description
+Alien Hunter is a 2003 American-Bulgarian science-fiction-thriller film
+directed by Ronald Krauss and starring James Spader, Carl Lewis and Leslie
+Stefanson.
+
+Synopsis: A former SETI cryptologist is invited to the South Pole to decipher
+an encrypted message that is possibly of extraterrestrial origin.
+
+:: Date
+2003-07-19
+
+:: References
+https://en.wikipedia.org/wiki/Alien_Hunter
+
+:: Ratings
+3 <paul>
+
+:: Major Themes
+extraterrestrial being [The story follows a team of researchers stationed in Antarctica as they investigate a strange artifact that is ultimately found to contain a living space alien.]
+alien brought pandemic [A main novelty of the story is that the alien was carrying a virus that was liable to destroy all life on earth.]
+
+:: Minor Themes
+Roswell UFO incident [A radio operator working in 1947 New Mexico received a strange signal originating from Roswell. Upon driving to the site of the signal, he found the alleged site of a crashed alien spaceship.]
+student and teacher romance [The college professor Julian revealed to his superior that he was involved with a female student.]
+infatuation [A foxy female student of the college professor Julian sent him an email with subject heading "SEX" and body text "I WANT YOU" - an email which Julian opened on his laptop toward the end of his lecture.]
+genetically modified food in society [Nyla was of the mind that the genetically modified, hydroponically grown crops being grown in the Antarctic research base greenhouse was the way of the future.]
+search for extraterrestrial intelligence [Julian had worked at SETI as a cryptologist.]
+coping with the death of a family member [Grisham kept a photo on his desk of his brother who'd died tragically in a flight training accident.]
+telepathic ability [The alien used telepathy to communicate with Julian.]
+nuclear weapons [The president ordered the firing of a limited nuclear device on the Antarctic research base to prevent the spread of a deadly alien virus.]
+facing death [The people at the Antarctic research base were informed they'd be nuked in three hours' time. One of the people resolved to fly out of there before the nuke arrived, while the others were more inclined to accept their fate with dignity.]
+the good of society vs. personal gain [There was a short deliberation among the alien virus exposed people at the Antarctic research base about whether they should leave before the nuke hit. If they left the base, they risked causing a pandemic; if they stayed, they'd die horribly in a nuclear conflagration.]
```

Comment:
How on Earth is there "personal gain" here??

Reply:
We used to have a "sacrifice to contain a pandemic" but perhaps it was generalised to https://www.themeontology.org/theme.php?name=sacrifice%20for%20one%27s%20people

Reply:
Well, you've definitely got a point there. I went with [sacrificing one's self to stop a contagion](https://www.themeontology.org/theme.php?name=sacrificing%20one%27s%20self%20to%20stop%20a%20contagion) since I found we have it.